src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -55,7 +55,9 @@ def items
       #
       # @return [Hash{String => Boolean}]
       def open_items
-        @open_items || {}
+        return default_open_items if !@open_items || @open_items.empty?
```

Comment:
Why is `default_open_items` called when `@open_items == {}`? I understand the nil case, but the empty one looks quite unexpected to me.